### ISSUE TITLE
System Integration - Impossible to define system-auth and system-syslog processors at the same time

### ISSUE THREAD

**Issue by herrBez** [elastic/integrations#4334]
In the system integration it is impossible to define a processors for the system.auth and the system.syslog datasets **at the same time**. 

When you try to store the integration with a processor for the system.auth and for the system.syslog, you receive this error message

<img width="206" alt="image" src="https://user-images.githubusercontent.com/6976484/192812681-abce153f-56f4-40cd-a73c-9404375a5190.png">

Steps to reproduce:

1. Create a new system integration
2. Add a processor in the system.auth dataset
3. Add a processor in the system.syslog dataset
4. Try to store the integration

**Comment by andrewkroh** [comment]
What was the YAML that you typed into the processors sections?

**Comment by herrBez** [comment]
Hi, I tried with several combination processors. For instance with one example of the [add field processor](https://www.elastic.co/guide/en/beats/filebeat/current/add-fields.html) documentation:

```yaml
- add_fields:
    target: project
    fields:
      name: myproject
      id: '574734885120952459'
```    

The problem seems related not to the content of the yaml (I even tried to use different processors), but rather by the fact that it is defined in both the system.syslog and the system.auth dataset.

**Comment by andrewkroh** [comment]
I can reproduce it under system package 1.19.3. The error is:

> `duplicated mapping key at line 10, column -111: processors: ^`

This is a screenshot of the config that I used:

![issue-4334-screenshot](https://user-images.githubusercontent.com/4565752/193079468-0a018271-c2f9-494c-9b93-6dfbbe4074dc.png)

**Comment by andrewkroh** [comment]
I think this was fixed in 1.19.5 via https://github.com/elastic/integrations/pull/4180.

**Comment by herrBez** [comment]
Thank you @andrewkroh.  I will close the issue
